test/lib/testMayaUsdCreateStageCommands.py

```diff
@@ -143,3 +143,39 @@ def testCreateStageWithCommand(self):
             cmds.redo()
         stageUfeSceneItem = ufe.Hierarchy.createItem(stageUfePath)
         self.assertIsNotNone(stageUfeSceneItem)
+
+    def testCreateStageNamedWorld(self):
+        # This test requires a Maya fix for the hidden world node.
+        # Without fix = '|world'
+        # With fix    = '|world|world'
+        worldPath = ufe.PathString.path('|world')
+        if str(worldPath) == '|world':
+            self.skipTest('Requires Maya for hidden world node.')
```

Comment:
Don't quite understand the message here: should it be
```suggestion
            self.skipTest('Requires Maya fix for hidden world node.')
```